packages/cloud_firestore/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.1
```

Comment:
If there is no public api change, the version should only be updated as a patch, so 0.9.0+2
https://www.dartlang.org/tools/pub/versioning#semantic-versions
Same for everything related in this PR.
@collinjackson @mklim can you confirm?

Reply:
Good catch, this is more of a bugfix and should just be bumping the patch numbers everywhere. [Sometimes it is appropriate to increment the minor version just based on the behavior of private code](https://semver.org/#spec-item-7) but I don't think this change really fits that edge case.

Reply:
I'm fine with doing this as 0.9.0+2